zeeguu/api/endpoints/teacher_dashboard/student.py

```diff
@@ -26,18 +26,23 @@
 @api.route("/basic_user_info/<id>", methods=["GET"])
 @requires_session
 def basic_user_info(id):
+    # This method doesn't seem to be used by the frontend
```

Comment:
really? then maybe we should remove it?

Reply:
![image](https://github.com/user-attachments/assets/bacb9194-9331-4d59-bf7d-bed001551c24)

Seems that way, I believe it was something that we didn't end up using when Iga did the onboarding.